l10n_th_withholding_tax_report/README.rst

```diff
@@ -45,20 +45,21 @@ To use this module, you need to:
 * View - Report type html.
   Export PDF - Report type pdf.
   Export XLSX - Report type xlsx.
+  Export TXT - Report type text.
 
 Changelog
```

Comment:
Thanks for seeing that the history log is not updated.
But normally, we want to keep the v12 history log too. So just add the new v13 log.